src/super_gradients/training/losses/__init__.py

```diff
@@ -1,3 +1,6 @@
+from super_gradients.common.registry.registry import LOSSES
+from super_gradients.common.object_names import Losses
+
```

Comment:
Note: This will not lead to circular loop and add backward compatibility if someone is importing Losses or LOSSES from this module.